packages/components/src/components/Snackbar/Snackbar.stories.tsx

```diff
@@ -34,27 +35,48 @@ export default meta
 
 type Story = StoryObj<SnackbarProps>
 
+const isWeb = Platform.OS !== 'web'
+
 const mobileComponentRenderer = (props: SnackbarProps) => {
-  const { isError, messageA11y, onActionPressed } = props.data || {}
+  const { isError, messageA11y, onActionPressed, offset } = props.data || {}
+  const { show } = useSnackbar()
   const onPressSnackbar = () => {
-    ShowSnackbar(props.message as string, {
+    show(props.message as string, {
       isError,
       messageA11y,
       onActionPressed,
+      offset,
     })
   }
 
-  return <Button label="Press for Snackbar" onPress={onPressSnackbar} />
+  return (
+    <>
+      <Button label="Press for Snackbar" onPress={onPressSnackbar} />
+    </>
+  )
+}
+
+export const _Default: Story = {
+  render: isWeb ? mobileComponentRenderer : undefined, // Render Snackbar flat in web
+  args: {
+    message: 'Message moved to Test Folder',
+    data: {
+      isError: false,
+      messageA11y: 'Message moved to Custom Folder with a11y',
+      onActionPressed: () => console.log('Action pressed'),
+    },
+  },
 }
 
-export const _Snackbar: Story = {
-  render: Platform.OS !== 'web' ? mobileComponentRenderer : undefined, // Render Snackbar flat in web
+export const __CustomOffset: Story = {
```

Comment:
Not sure about this being a separate story. Looking in Web, it's identical which is confusing as a demonstration tool.

Not sure if we can restrict this story to only appearing in mobile, otherwise we may be fine just having the Default story default a custom offset and updating as needed when testing on mobile devices.

Reply:
Hid this on web by adding a filter that will hide any stories on web that have a `'mobile-only'` tag

Reply:
Fancy 👍